sirepo/db_upgrade.py

```diff
@@ -79,6 +79,11 @@ def _20240322_remove_github_auth(qcall):
     qcall.auth_db.drop_table("auth_github_user_t")
 
 
+def _20240507_cloudmc_to_openmc(qcall):
+    for u in qcall.auth_db.all_uids():
+        _migrate_sim_type("cloudmc", "openmc", qcall, u)
+
```

Comment:
This line is failing for me with an error ```AttributeError: 'API' object has no attribute 'cookie'```.

I was on the master branch, created some cloudmc simulations, then checked out this branch and started ```sirepo service http``` and it failed when it was looking up the simulation_dir() during the db upgrade.